### DISCUSSION TITLE
Why is the statistics website loading so slowly?

### DISCUSSION THREAD

**Discussion by DimaSergeew** [plan-player-analytics/Plan#2618] [Q&A] [answered]
1) Sometimes I wait up to 5 minutes for the statistics site to load. Why is the web service loading so slowly?
2) I have a suggestion! I noticed it integrates the LiteBans plugin, so you can add the ability to see who and who banned, for example, in 7 days, 30 days, or 24 hours. Briefly set the ability to sort actions by time)
3) It would also be cool to integrate with CMI

**Comment by AuroraLS3** [comment, marked as answer]
Have you tried the frontend beta? (It's in the config) It should be loading a bit faster since the old front end loads more data at once.

Since everything is requested at once it can cause too much load at once, leading to longer times for the responses. In addition to this larger amounts of data take longer to analyze.

----

Date sorting might be possible to add to litebans, please open an issue for this.

----

CMI does not provide a proper API artifact for other projects to use it, so support for it will not be added. https://github.com/plan-player-analytics/Plan/issues/1430
